### PR TITLE
Netlify Build Preview - No more Tugboat

### PR DESCRIPTION
**Pull request by sherakama**
# READY FOR REVIEW

# Summary
- Removes Tugboat in favor of Netlify

# Needed By (Date)
- Whenever
- 
# Urgency
- Low

# Steps to Test

1. Review code
2. Review preview build

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -44,7 +45,6 @@
     "extra-watch-webpack-plugin": "^1.0",
     "file-loader": "^3.0",
     "filemanager-webpack-plugin": "^2.0",
-    "fsevents": "^2.1.3",
```
Comment (sherakama) [inline package.json:47]: fsevents is OSX only and does not work on netlify builds.

File: netlify.toml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+# ##############################################################################
+[[headers]]
+  for = "/*"
+  [headers.values]
+    Access-Control-Allow-Origin = "*"
+    Content-Security-Policy = "form-action https:"
+    X-Content-Type-Options = "nosniff"
+    Referrer-Policy = "origin-when-cross-origin"
+    Strict-Transport-Security = "max-age=2592000"
+    Permissions-Policy = "vibrate=(), geolocation=(), midi=(), notifications=(), push=(), microphone=(), camera=(), magnetometer=(), gyroscope=(), speaker=(), vibrate=(), fullscreen=()"
```
Comment (yvonnetangsu) [inline netlify.toml:22]: I'm wondering if we could enable fullscreen for testing those youtube embeds? Actually doesn't matter as much here in the Decanter repo, but I wouldn't mind having it back for the Giving repo for that purpose 🤔

**Review by yvonnetangsu — APPROVED** [review]
Preview build works! Yay!! 🥳  Thank you.

File: netlify.toml (hunk 2 above)
Comment (sherakama) [reply netlify.toml:22]: Ah yes. Probably can open a few of these up. These were just blanket turn off all the things.

**Review by sherakama — COMMENTED** [review]
Thanks

**Pull request merged**